ios/RCTMGL/RCTMGLMapViewManager.m

```diff
@@ -361,6 +363,16 @@ - (void)didTapMap:(UITapGestureRecognizer *)recognizer
             [self fireEvent:event withCallback:source.onPress];
             return;
         }
+    } else {
+        // So we got no features, maybe it was tapping on an on-screen button?
+        UIView *hitView = [recognizer.view hitTest:[recognizer locationInView:recognizer.view] withEvent:NULL];
+
+        // This is for empty taps, so we want to skip those
+        Class privateAnnotationContainerClass = NSClassFromString(@"MGLAnnotationContainerView");
```

Comment:
I'm assuming we need to check for this private class? maybe there should be an external header file for it instead if we want to merge this back upstream.